packages/api-clients/open-api/bffApi.yml

```diff
@@ -17340,6 +17340,62 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Problem"
+  /eservices/name/check:
```

Comment:
may we use a more restful path?

Reply:
what about `/eservices/names/availability`?